### PR TITLE
Update docs for demo

### LINKED ISSUE #109: Run the demo gives Go error
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same problem, before opening a new one.
- [X] This issue only relates to a single bug. I will open new issues for any other problems.

### Describe the bug

I followed the command described on https://developer.fyne.io/started/#run-the-demo

Since the text says "If you want to see the Fyne toolkit in action before you start to code your own application", I (wrongly) assumed I did not have to do / install anything first.

This is the output from my computer executed in folder 'temp':

➜  temp go run fyne.io/fyne/v2/cmd/fyne_demo
no required module provides package fyne.io/fyne/v2/cmd/fyne_demo: go.mod file not found in current directory or any parent directory; see 'go help modules'
➜  temp go version
go version go1.18 darwin/arm64

Later on I realised that the author meant that I had to do the steps described in Downloading section first. You need to take the phrase 'to code' literally.

However if you execute go mod tidy - it will remove the required fyne dependency since there is no Go file referencing it.
If you skip go mod tidy, so the dependency is there, you are also out of luck since there is no go.sum file.

So after repeating the steps in the Download section, I got the following error message:

➜  temp go run fyne.io/fyne/v2/cmd/fyne_demo
no required module provides package fyne.io/fyne/v2/cmd/fyne_demo; to add it:
	go get fyne.io/fyne/v2/cmd/fyne_demo

If you execute above go get command, it works.

A solution is to move the Installing section up, as first part of Running the demo.

### How to reproduce

see above

### Screenshots

_No response_

### Example code

n/a

### Fyne version

latest

### Go compiler version

1.18

### Operating system

macOS

### Operating system version

12.2.1

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by andydotxyz**
Splits it out to a new page as it's no longer as simple as installing from the current directory.

Fixes #109

### CODE REVIEW

File: started/demo.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+title: Run Fyne Demo
+
+---
+
+If you want to see the Fyne toolkit in action before you start to code your own application,
+you can see our [demo app](https://github.com/fyne-io/fyne/tree/master/cmd/fyne_demo) application.
+
+### Running
+
+If you want to, you can also install the demo using the following command (requires Go 1.16 or later):
```
Comment (Bluebugs) [inline started/demo.md:12]: I don't think that the command install fyne_demo, but just execute it.

File: started/demo.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: Run Fyne Demo
+
+---
+
+If you want to see the Fyne toolkit in action before you start to code your own application,
+you can see our [demo app](https://github.com/fyne-io/fyne/tree/master/cmd/fyne_demo) application.
```
Comment (Jacalz) [inline started/demo.md:8]: Having this worded "demo app application" seems a bit to me like it stutters. I think we should word that a bit different.

File: started/demo.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+By browsing the different tabs of the app you can see all the features of Fyne toolkit.
+
+### Installing
+
+It is possible to install the app as a graphical application like all others on your computer.
+We have the helpful `fyne` tool to do this for you.
+First you will need to install the tool:
+
+	go install fyne.io/fyne/v2/cmd/fyne@latest
```
Comment (Jacalz) [inline started/demo.md:28]: Should we be a bit more consistent with `$` at the beginning of commands?

**Review by Jacalz — CHANGES_REQUESTED** [review]
Well done. Just a few suggestions :)

File: started/demo.md (hunk 3 above)
Comment (andydotxyz) [reply started/demo.md:28]: Good catch. And in other pages they have been removed, so the ones where it is not present should be updated.
The codeblock style takes care of it - the note was that "$" gets in the way of copy/paste

**Comment by andydotxyz** [comment]
Thanks for reviving this @Jacalz - should be done now

**Review by Jacalz — APPROVED** [review]
Nicely done :)

**Pull request merged**